parlai/core/metrics.py

```diff
@@ -31,11 +31,10 @@
     nltkbleu = None
 
 try:
-    import rouge as rouge
+    import rouge
 except ImportError:
     # User doesn't have rouge installed, so we can't use it for rouge
     # We'll just turn off things, but we might want to warn the user
```

Comment:
prob should remove "we might want to warn the user" since we are no longer warning them 🌵